modules/cert-manager-istio-creating-issuer.adoc

```diff
@@ -13,7 +13,14 @@ Use this procedure to create the root CA issuer for Istio-CSR agent.
 Other supported issuers can be used, except for the ACME issuer, which is not supported. For more information, see "{cert-manager-operator} issuer providers".
 ====
 
-* Create a YAML file, for example, `issuer.yaml`, that defines the `Issuer` and `Certificate` objects:
+. Create a new project for installing Istio-CSR by running the following command. You can use an existing project and skip this step.
++ 
+[source,terminal] 
+---- 
+$ oc new-project <istio_csr_project_name> 
+---- 
+
+. Create a YAML file, for example, `issuer.yaml`, that defines the `Issuer` and `Certificate` objects:
```

Comment:
Is it important to call it `issuer.yaml` at all? You could avoid the "for example" clause if it's not. 

E.g., "Create a YAML file that defines..." followed by "Example issuer file" as a title.